katello/katello-backup

```diff
@@ -7,14 +7,17 @@ require "yaml"
 
 @options = {}
 @dir = nil
+@is_foreman_proxy_content = !`rpm -qa | grep foreman-proxy-content`.empty?
+@databases = ['pulp', 'postgres', 'mongodb']
```

Comment:
@ehelms configs are always included in every backup, so they weren't included in this array